data/net/minecraft/client/renderer/RenderType.mapping

```diff
@@ -48,35 +73,128 @@ CLASS net/minecraft/client/renderer/RenderType
 		ARG 0 location
 	METHOD itemEntityTranslucentCull (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
 		ARG 0 location
+	METHOD lambda$static$0 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$1 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$10 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$11 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$12 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$13 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$14 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$15 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$16 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$17 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$18 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$19 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$2 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$20 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$21 (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+	METHOD lambda$static$3 (Lnet/minecraft/resources/ResourceLocation;Ljava/lang/Boolean;)Lnet/minecraft/client/renderer/RenderType;
+		ARG 0 id
+		ARG 1 cull
```

Comment:
This is not cull, it just affects whether the rendering affects the outline of the object.